distribution/docker/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
```

Comment:
please add a comment that indicates this file is only used for testing of the docker image, not what we publish